main.js

```diff
@@ -1,30 +1,47 @@
-const studentList =[];
-const studentName = document.querySelector('#studentName');
-const houses = [
-  'Gryffindor', 
-  'Hufflepuff', 
-  'Ravenclaw', 
-  'Slytherin'
-];
-const startBtn = document.querySelector('#startBtn');
-const sortBtn = document.querySelector('#sortBtn');
-
+const studentList = [];
+const houses = ["Gryffindor", "Hufflepuff", "Ravenclaw", "Slytherin"];
+const startBtn = document.querySelector("#startBtn");
+const sortBtn = document.querySelector("#sortBtn");
 
 const printToDom = (divId, textToPrint) => {
   const selectedDiv = document.querySelector(divId);
   selectedDiv.innerHTML = textToPrint;
-}
-
-const hideForm = () => {
-  document.querySelector('#studentForm').style.display = 'none';
 };
 
-const showForm = () => {
-  document.getElementById('#studentForm').style.display = 'block';
+const showForm = (e) => {
+  const buttonId = e.target.id;
+
+  if (buttonId === "startBtn") {
+    document.querySelector("#studentForm").innerHTML = `<div id="studentForm">
+    <form>
+      <h1>Enter First Year's Name</h1>
+      <div class="row mb-3">
+      <label for="student-name" class="col-sm-2 col-form-label">Student:</label>
+      <div class="col-sm-10">
+        <input type="text" class="form-control d-inline" id="studentName" required>
+      </div>
+      </div>
+      <button type="button" class="btn btn-secondary" id="sortBtn">Sort!</button>
+    </form>
+  </div>`;
+  }
+  document.querySelector("#sortBtn").addEventListener("click", formContent);
 };
 
+// const formAlert = () => {
```

Comment:
Make sure that code on the main (production) branch does not have commented out code in it. I know you are still working on this but just as a reminder to remove any unused code before going live on main.